Not working with yarn

When trying to install as a global dependency with yarn package do not work

```
$ elm-app --help
module.js:327
    throw err;
    ^

Error: Cannot find module '../node_modules/elm/package.json'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/fnez/.config/yarn/global/node_modules/create-elm-app/bin/elm-app-cli.js:9:28)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Function.Module.runMain (module.js:441:10)
```

--- a/bin/create-elm-app-cli.js
+++ b/bin/create-elm-app-cli.js
@@ -4,7 +4,7 @@ const path = require('path');
 const spawn = require('cross-spawn');
 const argv = require('minimist')(process.argv.slice(2));
 const version = require('../package.json').version;
-const elmPlatformVersion = require('../node_modules/elm/package.json').version;
+const elmPlatformVersion = require('elm/package.json').version;
 const commands = argv._;
 
 if (commands.length === 0) {

